internal/api/server.go

```diff
@@ -85,7 +86,8 @@ func NewServer(configFile string) (*http.Server, error) {
 
 	env := &Environment{}
 	env.DB = db
-	router := NewGoCertRouter(env)
+	metricsHandler := metrics.NewPrometheusHandler()
```

Comment:
I don't think we need to make any change in this file

Reply:
Done